contracts/utils/ChainportLERC20.sol

```diff
@@ -1,308 +1,1375 @@
+    function transfer(address recipient, uint256 amount) external returns (bool);
+
+    /**
+     * @dev Returns the remaining number of tokens that `spender` will be
+     * allowed to spend on behalf of `owner` through {transferFrom}. This is
+     * zero by default.
+     *
+     * This value changes when {approve} or {transferFrom} are called.
+     */
+    function allowance(address owner, address spender) external view returns (uint256);
+
+    /**
+     * @dev Sets `amount` as the allowance of `spender` over the caller's tokens.
+     *
+     * Returns a boolean value indicating whether the operation succeeded.
+     *
+     * IMPORTANT: Beware that changing an allowance with this method brings the risk
+     * that someone may use both the old and the new allowance by unfortunate
+     * transaction ordering. One possible solution to mitigate this race
+     * condition is to first reduce the spender's allowance to 0 and set the
+     * desired value afterwards:
+     * https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729
+     *
+     * Emits an {Approval} event.
+     */
+    function approve(address spender, uint256 amount) external returns (bool);
+
+    /**
+     * @dev Moves `amount` tokens from `sender` to `recipient` using the
+     * allowance mechanism. `amount` is then deducted from the caller's
+     * allowance.
+     *
+     * Returns a boolean value indicating whether the operation succeeded.
+     *
+     * Emits a {Transfer} event.
+     */
+    function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
+
+    /**
+     * @dev Emitted when `value` tokens are moved from one account (`from`) to
+     * another (`to`).
+     *
+     * Note that `value` may be zero.
+     */
+    event Transfer(address indexed from, address indexed to, uint256 value);
+
+    /**
+     * @dev Emitted when the allowance of a `spender` for an `owner` is set by
+     * a call to {approve}. `value` is the new allowance.
+     */
+    event Approval(address indexed owner, address indexed spender, uint256 value);
+}
+
+
+// File contracts/BridgeMintableTokenV2.sol
+
+pragma solidity 0.6.12;
+
+
+
+/**
+ * BridgeMintableToken contract V2.
+ * @author Marko Lazic
+ * Date created: 7.10.21.
+ * Github: markolazic01
+ */
+contract BridgeMintableTokenV2 is ERC20Upgradeable, PausableUpgradeable {
+
+    address public chainportBridge;
+    address public chainportCongress;
+    mapping(address => bool) public isBlocked;
+
+    event Mint(address indexed to, uint256 amount);
+    event AddressBlocked(address indexed addr);
+    event AddressUnblocked(address indexed addr);
+    event ChainportBridgeChanged(address indexed addr);
+    event ChainportCongressChanged(address indexed addr);
+    event StuckTokenWithdrawn(address indexed token, address indexed to, uint256 amount);
+    event BlackFundsDestroyed(address indexed token, uint256 amount);
+
+    /// Lossless Compliance
+    address public admin;
```

Comment:
recoveryAdmin and related functionality is missing.

Reply:
Added on commit [a52c6cb](https://github.com/Lossless-Cash/lossless-v3/pull/105/commits/a52c6cb4c5e2dd25e1340e0d74add15e8bfa0ea9)